recipes/ollama/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "ollama" %}
+{% set goname = "github.com/jmorganca/ollama" %}
+{% set version = "0.1.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  #url: https://{{ goname }}/archive/v{{ version }}.tar.gz
+  #sha256: 881d81d6a113f5361fc1648ee3f07f596b96a25ff9c9e6b243c2386adb8a0b26
+  git_url: https://github.com/jmorganca/ollama.git
+  git_rev: v{{ version }}
+  git_depth: 1
+
+build:
+  number: 0
+  skip: True  # [win]
+  script:
+    - export GOFLAGS="'-ldflags=-X=github.com/jmorganca/ollama/version.Version={{ version }} -X=github.com/jmorganca/ollama/server.mode=release'"
+    - go generate ./...
+    - go install .
+    - go-licenses save --save_path licenses ./...
+
+requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
```

Ensure CUDA compiler is only added if cuda_compiler_version is set

```suggestion
    - {{ compiler('cuda') }}  # [cuda_compiler_version not in (undefined, "None")]
```